projects/plugins/crm/includes/ZeroBSCRM.AJAX.php

```diff
@@ -794,7 +795,7 @@ function ZeroBSCRM_get_quote_template() {
 				$quote_val = '[QUOTEVALUE]';
 			}
 			if ( empty( $quote_date ) ) {
-				$quote_date = gmdate( 'Y-m-d' );
+				$quote_date = wp_date( get_option( 'date_format' ) );
```

Comment:
In general I'd recommend using `jpcrm_uts_to_date_str( $uts )`, but this appears to be equally valid.

Reply:
Yes I think when we review all timezone functions generally we might want to swap things out if needed.

Reply:
Following up, I ow gone through and change these due to the timezone issues.